File: week_3/1-money_change/change.hs
-- by Kirill Elagin

get_change :: Int -> Int
get_change 0 = 0
get_change m  
  | m > 9 = 1 + get_change(m-10)
  | m > 4 = 1 + get_change(m-5)
  | m > 0 = 1 + get_change(m-1)



-- write your code here
-- get_change  = 

main :: IO ()
main = do
  [m] <- fmap words getLine
  print $ get_change (read m)

File: week_3/1-money_change/change.py
# Uses python3
import sys

def get_change(m):
    amount = 0
    while(m > 9):
        amount = amount + 1
        m = m -10
    while(m > 4):
        amount = amount + 1
        m = m - 5
    while(m != 0):
        amount = amount + 1
        m = m - 1

    #write your code here
    return amount

if __name__ == '__main__':
#    input = sys.stdin.read()
    val = input("Enter value: ")
    n = int(val)
    print(get_change(n))
